static/js/components/NewDropDownSearchBar.jsx

```diff
@@ -0,0 +1,55 @@
+// NewDropDownsearchBar.jsx
+import React from "react";
+import PropTypes from "prop-types"; // Import PropTypes for prop validation
+import Autocomplete from "@mui/material/Autocomplete";
+import TextField from "@mui/material/TextField";
+
+const NewDropDownSearchBar = ({
```

Comment:
As discussed on the PR that was closed, can you make that component generic, i.e. not call the parameters instrumentList and selectInstrument, but rather something like optionsList and selectOption? As well as make the label of the TextField a parameter and not hardcode it to "Select an instrument".

Once you are done with that, and it works as expected, happy to merge.